<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Iker Rivas-González</title> <meta name="author" content="Iker Rivas-González"> <meta name="description" content="Postdoctoral researcher with experience in evolutionary and population genetics, data science and programming. "> <meta name="keywords" content="bioinformatics, population-genetics, primates"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://rivasiker.github.io//publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/">Iker Rivas-González</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories</a> </li> <li class="nav-item "> <a class="nav-link" href="/news/">news</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="nav-item "> <a class="nav-link" href="/contact/">contact</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="year">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#bd2736"><a href="https://www.biorxiv.org" rel="external nofollow noopener" target="_blank">bioRxiv</a></abbr></div> <div id="yoo2024apet2t" class="col-sm-8"> <div class="title">Complete sequencing of ape genomes</div> <div class="author"> DongAhn Yoo, Arang Rhie, Prajna Hebbar, and <span class="more-authors" title="click to view 120 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '120 more authors' ? 'Francesca Antonacci, Glennis A. Logsdon, Steven J. Solar, Dmitry Antipov, Brandon D. Pickett, Yana Safonova, Francesco Montinaro, Yanting Luo, Joanna Malukiewicz, Jessica M. Storer, Jiadong Lin, Abigail N. Sequeira, Riley J. Mangan, Glenn Hickey, Graciela Monfort Anez, Parithi Balachandran, Anton Bankevich, Christine R. Beck, Arjun Biddanda, Matthew Borchers, Gerard G. Bouffard, Emry Brannan, Shelise Y. Brooks, Lucia Carbone, Laura Carrel, Agnes P. Chan, Juyun Crawford, Mark Diekhans, Eric Engelbrecht, Cedric Feschotte, Giulio Formenti, Gage H. Garcia, Luciana de Gennaro, David Gilbert, Richard E. Green, Andrea Guarracino, Ishaan Gupta, Diana Haddad, Junmin Han, Robert S. Harris, Gabrielle A. Hartley, William T. Harvey, Michael Hiller, Kendra Hoekzema, Marlys L. Houck, Hyeonsoo Jeong, Kaivan Kamali, Manolis Kellis, Bryce Kille, Chul Lee, Youngho Lee, William Lees, Alexandra P. Lewis, Qiuhui Li, Mark Loftus, Yong Hwee Eddie Loh, Hailey Loucks, Jian Ma, Yafei Mao, Juan F. I. Martinez, Patrick Masterson, Rajiv C. McCoy, Barbara McGrath, Sean McKinney, Britta S. Meyer, Karen H. Miga, Saswat K. Mohanty, Katherine M. Munson, Karol Pal, Matt Pennell, Pavel A. Pevzner, David Porubsky, Tamara Potapova, Francisca R. Ringeling, Joana L. Rocha, Oliver A. Ryder, Samuel Sacco, Swati Saha, Takayo Sasaki, Michael C. Schatz, Nicholas J. Schork, Cole Shanks, Linnéa Smeds, Dongmin R. Son, Cynthia Steiner, Alexander P. Sweeten, Michael G. Tassia, Françoise Thibaud-Nissen, Edmundo Torres-González, Mihir Trivedi, Wenjie Wei, Julie Wertz, Muyu Yang, Panpan Zhang, Shilong Zhang, Yang Zhang, Zhenmiao Zhang, Sarah A. Zhao, Yixin Zhu, Erich D. Jarvis, Jennifer L. Gerton, Iker Rivas-González, Benedict Paten, Zachary A. Szpiech, Christian D. Huber, Tobias L. Lenz, Miriam K. Konkel, Soojin V. Yi, Stefan Canzar, Corey T. Watson, Peter H. Sudmant, Erin Molloy, Erik Garrison, Craig B. Lowe, Mario Ventura, Rachel J. O’Neill, Sergey Koren, Kateryna D. Makova, Adam M. Phillippy, Evan E. Eichler' : '120 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '0'); ">120 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em> 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.biorxiv.org/content/early/2024/07/31/2024.07.31.605654" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p>We present haplotype-resolved reference genomes and comparative analyses of six ape species, namely: chimpanzee, bonobo, gorilla, Bornean orangutan, Sumatran orangutan, and siamang. We achieve chromosome-level contiguity with unparalleled sequence accuracy (&lt;1 error in 500,000 base pairs), completely sequencing 215 gapless chromosomes telomere-to-telomere. We resolve challenging regions, such as the major histocompatibility complex and immunoglobulin loci, providing more in-depth evolutionary insights. Comparative analyses, including human, allow us to investigate the evolution and diversity of regions previously uncharacterized or incompletely studied without bias from mapping to the human reference. This includes newly minted gene families within lineage-specific segmental duplications, centromeric DNA, acrocentric chromosomes, and subterminal heterochromatin. This resource should serve as a definitive baseline for all future evolutionary studies of humans and our closest living ape relatives.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">yoo2024apet2t</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Yoo, DongAhn and Rhie, Arang and Hebbar, Prajna and Antonacci, Francesca and Logsdon, Glennis A. and Solar, Steven J. and Antipov, Dmitry and Pickett, Brandon D. and Safonova, Yana and Montinaro, Francesco and Luo, Yanting and Malukiewicz, Joanna and Storer, Jessica M. and Lin, Jiadong and Sequeira, Abigail N. and Mangan, Riley J. and Hickey, Glenn and Anez, Graciela Monfort and Balachandran, Parithi and Bankevich, Anton and Beck, Christine R. and Biddanda, Arjun and Borchers, Matthew and Bouffard, Gerard G. and Brannan, Emry and Brooks, Shelise Y. and Carbone, Lucia and Carrel, Laura and Chan, Agnes P. and Crawford, Juyun and Diekhans, Mark and Engelbrecht, Eric and Feschotte, Cedric and Formenti, Giulio and Garcia, Gage H. and Gennaro, Luciana de and Gilbert, David and Green, Richard E. and Guarracino, Andrea and Gupta, Ishaan and Haddad, Diana and Han, Junmin and Harris, Robert S. and Hartley, Gabrielle A. and Harvey, William T. and Hiller, Michael and Hoekzema, Kendra and Houck, Marlys L. and Jeong, Hyeonsoo and Kamali, Kaivan and Kellis, Manolis and Kille, Bryce and Lee, Chul and Lee, Youngho and Lees, William and Lewis, Alexandra P. and Li, Qiuhui and Loftus, Mark and Loh, Yong Hwee Eddie and Loucks, Hailey and Ma, Jian and Mao, Yafei and Martinez, Juan F. I. and Masterson, Patrick and McCoy, Rajiv C. and McGrath, Barbara and McKinney, Sean and Meyer, Britta S. and Miga, Karen H. and Mohanty, Saswat K. and Munson, Katherine M. and Pal, Karol and Pennell, Matt and Pevzner, Pavel A. and Porubsky, David and Potapova, Tamara and Ringeling, Francisca R. and Rocha, Joana L. and Ryder, Oliver A. and Sacco, Samuel and Saha, Swati and Sasaki, Takayo and Schatz, Michael C. and Schork, Nicholas J. and Shanks, Cole and Smeds, Linn{\'e}a and Son, Dongmin R. and Steiner, Cynthia and Sweeten, Alexander P. and Tassia, Michael G. and Thibaud-Nissen, Fran{\c c}oise and Torres-Gonz{\'a}lez, Edmundo and Trivedi, Mihir and Wei, Wenjie and Wertz, Julie and Yang, Muyu and Zhang, Panpan and Zhang, Shilong and Zhang, Yang and Zhang, Zhenmiao and Zhao, Sarah A. and Zhu, Yixin and Jarvis, Erich D. and Gerton, Jennifer L. and Rivas-Gonz{\'a}lez, Iker and Paten, Benedict and Szpiech, Zachary A. and Huber, Christian D. and Lenz, Tobias L. and Konkel, Miriam K. and Yi, Soojin V. and Canzar, Stefan and Watson, Corey T. and Sudmant, Peter H. and Molloy, Erin and Garrison, Erik and Lowe, Craig B. and Ventura, Mario and O{\textquoteright}Neill, Rachel J. and Koren, Sergey and Makova, Kateryna D. and Phillippy, Adam M. and Eichler, Evan E.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Complete sequencing of ape genomes}</span><span class="p">,</span>
  <span class="na">elocation-id</span> <span class="p">=</span> <span class="s">{2024.07.31.605654}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2024.07.31.605654}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Laboratory}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioRxiv}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#2db361"><a href="https://academic.oup.com/emph" rel="external nofollow noopener" target="_blank">EMPH</a></abbr></div> <div id="rivas-gonzalez2024comparative" class="col-sm-8"> <div class="title">A multi-million-year natural experiment: comparative genomics on a massive scale and its implications for human health</div> <div class="author"> <em>Iker Rivas-González</em>, and Jenny Tung</div> <div class="periodical"> <em>Evolution, Medicine, and Public Health</em> 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1093/emph/eoae006" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p>Improving the diversity and quality of genome assemblies for non-human mammals has been a long-standing goal of comparative genomics. The last year saw substantial progress towards this goal, including the release of genome alignments for 240 mammals and nearly half the primate order. These resources have increased our ability to identify evolutionarily constrained regions of the genome, and together strongly support the importance of these regions to biomedically relevant trait variation in humans. They also provide new strategies for identifying the genetic basis of changes unique to individual lineages, illustrating the value of evolutionary comparative approaches for understanding human health</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">rivas-gonzalez2024comparative</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Rivas-González, Iker and Tung, Jenny}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{A multi-million-year natural experiment: comparative genomics on a massive scale and its implications for human health}}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Evolution, Medicine, and Public Health}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{eoae006}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2050-6201}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/emph/eoae006}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#0075bc"><a href="https://www.pnas.org" rel="external nofollow noopener" target="_blank">PNAS</a></abbr></div> <div id="mirarab2024region" class="col-sm-8"> <div class="title">A region of suppressed recombination misleads neoavian phylogenomics</div> <div class="author"> Siavash Mirarab, <em>Iker Rivas-González</em>, Shaohong Feng, and <span class="more-authors" title="click to view 16 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '16 more authors' ? 'Josefin Stiller, Qi Fang, Uyen Mai, Glenn Hickey, Guangji Chen, Nadolina Brajuka, Olivier Fedrigo, Giulio Formenti, Jochen B. W. Wolf, Kerstin Howe, Agostinho Antunes, Mikkel H. Schierup, Benedict Paten, Erich D. Jarvis, Guojie Zhang, Edward L. Braun' : '16 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '0'); ">16 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences</em> 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://pnas.org/doi/10.1073/pnas.2319506121" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p>Genomes are typically mosaics of regions with different evolutionary histories. When speciation events are closely spaced in time, recombination makes the regions sharing the same history small, and the evolutionary history changes rapidly as we move along the genome. When examining rapid radiations such as the early diversification of Neoaves 66 Mya, typically no consistent history is observed across segments exceeding kilobases of the genome. Here, we report an exception. We found that a 21-Mb region in avian genomes, mapped to chicken chromosome 4, shows an extremely strong and discordance-free signal for a history different from that of the inferred species tree. Such a strong discordance-free signal, indicative of suppressed recombination across many millions of base pairs, is not observed elsewhere in the genome for any deep avian relationships. Although long regions with suppressed recombination have been documented in recently diverged species, our results pertain to relationships dating circa 65 Mya. We provide evidence that this strong signal may be due to an ancient rearrangement that blocked recombination and remained polymorphic for several million years prior to fixation. We show that the presence of this region has misled previous phylogenomic efforts with lower taxon sampling, showing the interplay between taxon and locus sampling. We predict that similar ancient rearrangements may confound phylogenetic analyses in other clades, pointing to a need for new analytical models that incorporate the possibility of such events.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mirarab2024region</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{121}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0027-8424, 1091-6490}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1073/pnas.2319506121}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2024-04-02}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proceedings of the National Academy of Sciences}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mirarab, Siavash and Rivas-González, Iker and Feng, Shaohong and Stiller, Josefin and Fang, Qi and Mai, Uyen and Hickey, Glenn and Chen, Guangji and Brajuka, Nadolina and Fedrigo, Olivier and Formenti, Giulio and Wolf, Jochen B. W. and Howe, Kerstin and Antunes, Agostinho and Schierup, Mikkel H. and Paten, Benedict and Jarvis, Erich D. and Zhang, Guojie and Braun, Edward L.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e2319506121}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#000000"><a href="https://www.nature.com" rel="external nofollow noopener" target="_blank">Nature</a></abbr></div> <div id="stiller2024avian" class="col-sm-8"> <div class="title">Complexity of avian evolution revealed by family-level genomes</div> <div class="author"> Josefin Stiller, Shaohong Feng, Al-Aabid Chowdhury, and <span class="more-authors" title="click to view 49 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '49 more authors' ? 'Iker Rivas-González, David A. Duchêne, Qi Fang, Yuan Deng, Alexey Kozlov, Alexandros Stamatakis, Santiago Claramunt, Jacqueline M. T. Nguyen, Simon Y. W. Ho, Brant C. Faircloth, Julia Haag, Peter Houde, Joel Cracraft, Metin Balaban, Uyen Mai, Guangji Chen, Rongsheng Gao, Chengran Zhou, Yulong Xie, Zijian Huang, Zhen Cao, Zhi Yan, Huw A. Ogilvie, Luay Nakhleh, Bent Lindow, Benoit Morel, Jon Fjeldså, Peter A. Hosner, Rute R. Fonseca, Bent Petersen, Joseph A. Tobias, Tamás Székely, Jonathan David Kennedy, Andrew Hart Reeve, Andras Liker, Martin Stervander, Agostinho Antunes, Dieter Thomas Tietze, Mads Bertelsen, Fumin Lei, Carsten Rahbek, Gary R. Graves, Mikkel H. Schierup, Tandy Warnow, Edward L. Braun, M. Thomas P. Gilbert, Erich D. Jarvis, Siavash Mirarab, Guojie Zhang' : '49 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '0'); ">49 more authors</span> </div> <div class="periodical"> <em>Nature</em> 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1038/s41586-024-07323-1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p>Despite tremendous efforts in the past decades, relationships among main avian lineages remain heavily debated without a clear resolution. Discrepancies have been attributed to diversity of species sampled, phylogenetic method, and the choice of genomic regions 1–3. Here, we address these issues by analyzing genomes of 363 bird species 4 (218 taxonomic families, 92% of total). Using intergenic regions and coalescent methods, we present a well-supported tree but also a remarkable degree of discordance. The tree confirms that Neoaves experienced rapid radiation at or near the Cretaceous–Paleogene (K–Pg) boundary. Sufficient loci rather than extensive taxon sampling were more effective in resolving difficult nodes. Remaining recalcitrant nodes involve species that challenge modeling due to extreme GC content, variable substitution rates, incomplete lineage sorting, or complex evolutionary events such as ancient hybridization. Assessment of the impacts of different genomic partitions showed high heterogeneity across the genome. We discovered sharp increases in effective population size, substitution rates, and relative brain size following the K–Pg extinction event, supporting the hypothesis that emerging ecological opportunities catalyzed the diversification of modern birds. The resulting phylogenetic estimate offers novel insights into the rapid radiation of modern birds and provides a taxon-rich backbone tree for future comparative studies.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">stiller2024avian</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Complexity of avian evolution revealed by family-level genomes}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1476-4687}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-024-07323-1}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Stiller, Josefin and Feng, Shaohong and Chowdhury, Al-Aabid and Rivas-González, Iker and Duchêne, David A. and Fang, Qi and Deng, Yuan and Kozlov, Alexey and Stamatakis, Alexandros and Claramunt, Santiago and Nguyen, Jacqueline M. T. and Ho, Simon Y. W. and Faircloth, Brant C. and Haag, Julia and Houde, Peter and Cracraft, Joel and Balaban, Metin and Mai, Uyen and Chen, Guangji and Gao, Rongsheng and Zhou, Chengran and Xie, Yulong and Huang, Zijian and Cao, Zhen and Yan, Zhi and Ogilvie, Huw A. and Nakhleh, Luay and Lindow, Bent and Morel, Benoit and Fjeldså, Jon and Hosner, Peter A. and da Fonseca, Rute R. and Petersen, Bent and Tobias, Joseph A. and Székely, Tamás and Kennedy, Jonathan David and Reeve, Andrew Hart and Liker, Andras and Stervander, Martin and Antunes, Agostinho and Tietze, Dieter Thomas and Bertelsen, Mads and Lei, Fumin and Rahbek, Carsten and Graves, Gary R. and Schierup, Mikkel H. and Warnow, Tandy and Braun, Edward L. and Gilbert, M. Thomas P. and Jarvis, Erich D. and Mirarab, Siavash and Zhang, Guojie}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#263091"><a href="https://www.sciencedirect.com/journal/theoretical-population-biology" rel="external nofollow noopener" target="_blank">TPB</a></abbr></div> <div id="hobolth2024phasetype" class="col-sm-8"> <div class="title">Phase-type distributions in mathematical population genetics: An emerging framework</div> <div class="author"> Asger Hobolth, <em>Iker Rivas-González</em>, Mogens Bladt, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Andreas Futschik' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '0'); ">1 more author</span> </div> <div class="periodical"> <em>Theoretical Population Biology</em> 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0040580924000212" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p>A phase-type distribution is the time to absorption in a continuous- or discrete-time Markov chain. Phase-type distributions can be used as a general framework to calculate key properties of the standard coalescent model and many of its extensions. Here, the ‘phases’ in the phase-type distribution correspond to states in the ancestral process. For example, the time to the most recent common ancestor and the total branch length are phase-type distributed. Furthermore, the site frequency spectrum follows a multivariate discrete phase-type distribution and the joint distribution of total branch lengths in the two-locus coalescent-with-recombination model is multivariate phase-type distributed. In general, phase-type distributions provide a powerful mathematical framework for coalescent theory because they are analytically tractable using matrix manipulations. The purpose of this review is to explain the phase-type theory and demonstrate how the theory can be applied to derive basic properties of coalescent models. These properties can then be used to obtain insight into the ancestral process, or they can be applied for statistical inference. In particular, we show the relation between classical first-step analysis of coalescent models and phase-type calculations. We also show how reward transformations in phase-type theory lead to easy calculation of covariances and correlation coefficients between e.g. tree height, tree length, external branch length, and internal branch length. Furthermore, we discuss how these quantities can be used for statistical inference based on estimating equations. Providing an alternative to previous work based on the Laplace transform, we derive likelihoods for small-size coalescent trees based on phase-type theory. Overall, our main aim is to demonstrate that phase-type distributions provide a convenient general set of tools to understand aspects of coalescent models that are otherwise difficult to derive. Throughout the review, we emphasize the versatility of the phase-type framework, which is also illustrated by our accompanying R-code. All our analyses and figures can be reproduced from code available on GitHub.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">hobolth2024phasetype</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Phase-type distributions in mathematical population genetics: An emerging framework}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Theoretical Population Biology}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0040-5809}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.tpb.2024.03.001}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Hobolth, Asger and Rivas-González, Iker and Bladt, Mogens and Futschik, Andreas}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Coalescent, Laplace transform, Likelihood inference, Phase–type theory, Population genetics, Reward transformation}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#037165"><a href="https://journals.plos.org/plosgenetics/" rel="external nofollow noopener" target="_blank">PLoS Genet</a></abbr></div> <div id="rivas-gonzalez2024trails" class="col-sm-8"> <div class="title">TRAILS: Tree reconstruction of ancestry using incomplete lineage sorting</div> <div class="author"> <em>Iker Rivas-González</em>, Mikkel H. Schierup, John Wakeley, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Asger Hobolth' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '0'); ">1 more author</span> </div> <div class="periodical"> <em>PLOS Genetics</em> 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1371/journal.pgen.1010836" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p>Genome-wide genealogies of multiple species carry detailed information about demographic and selection processes on individual branches of the phylogeny. Here, we introduce TRAILS, a hidden Markov model that accurately infers time-resolved population genetics parameters, such as ancestral effective population sizes and speciation times, for ancestral branches using a multi-species alignment of three species and an outgroup. TRAILS leverages the information contained in incomplete lineage sorting fragments by modelling genealogies along the genome as rooted three-leaved trees, each with a topology and two coalescent events happening in discretized time intervals within the phylogeny. Posterior decoding of the hidden Markov model can be used to infer the ancestral recombination graph for the alignment and details on demographic changes within a branch. Since TRAILS performs posterior decoding at the base-pair level, genome-wide scans based on the posterior probabilities can be devised to detect deviations from neutrality. Using TRAILS on a human-chimp-gorilla-orangutan alignment, we recover speciation parameters and extract information about the topology and coalescent times at high resolution.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">rivas-gonzalez2024trails</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{TRAILS: Tree reconstruction of ancestry using incomplete lineage sorting}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{20}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1553-7404}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pgen.1010836}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS Genetics}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Public Library of Science (PLoS)}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Rivas-Gonz{\'a}lez, Iker and Schierup, Mikkel H. and Wakeley, John and Hobolth, Asger}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Palamara, Pier Francesco}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e1010836}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="year">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#C72326"><a href="https://www.science.org/journal/science" rel="external nofollow noopener" target="_blank">Science</a></abbr></div> <div id="shao2023phylogenomic" class="col-sm-8"> <div class="title">Phylogenomic analyses provide insights into primate evolution</div> <div class="author"> Yong Shao, Long Zhou, Fang Li, and <span class="more-authors" title="click to view 40 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '40 more authors' ? 'Lan Zhao, Bao-Lin Zhang, Feng Shao, Jia-Wei Chen, Chun-Yan Chen, Xupeng Bi, Xiao-Lin Zhuang, Hong-Liang Zhu, Jiang Hu, Zongyi Sun, Xin Li, Depeng Wang, Iker Rivas-González, Sheng Wang, Yun-Mei Wang, Wu Chen, Gang Li, Hui-Meng Lu, Yang Liu, Lukas F. K. Kuderna, Kyle Kai-How Farh, Peng-Fei Fan, Li Yu, Ming Li, Zhi-Jin Liu, George P. Tiley, Anne D. Yoder, Christian Roos, Takashi Hayakawa, Tomas Marques-Bonet, Jeffrey Rogers, Peter D. Stenson, David N. Cooper, Mikkel Heide Schierup, Yong-Gang Yao, Ya-Ping Zhang, Wen Wang, Xiao-Guang Qi, Guojie Zhang, Dong-Dong Wu' : '40 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '0'); ">40 more authors</span> </div> <div class="periodical"> <em>Science</em> 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.science.org/doi/abs/10.1126/science.abn6919" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p> Comparative analysis of primate genomes within a phylogenetic context is essential for understanding the evolution of human genetic architecture and primate diversity. We present such a study of 50 primate species spanning 38 genera and 14 families, including 27 genomes first reported here, with many from previously less well represented groups, the New World monkeys and the Strepsirrhini. Our analyses reveal heterogeneous rates of genomic rearrangement and gene evolution across primate lineages. Thousands of genes under positive selection in different lineages play roles in the nervous, skeletal, and digestive systems and may have contributed to primate innovations and adaptations. Our study reveals that many key genomic innovations occurred in the Simiiformes ancestral node and may have had an impact on the adaptive radiation of the Simiiformes and human evolution. </p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shao2023phylogenomic</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shao, Yong and Zhou, Long and Li, Fang and Zhao, Lan and Zhang, Bao-Lin and Shao, Feng and Chen, Jia-Wei and Chen, Chun-Yan and Bi, Xupeng and Zhuang, Xiao-Lin and Zhu, Hong-Liang and Hu, Jiang and Sun, Zongyi and Li, Xin and Wang, Depeng and Rivas-González, Iker and Wang, Sheng and Wang, Yun-Mei and Chen, Wu and Li, Gang and Lu, Hui-Meng and Liu, Yang and Kuderna, Lukas F. K. and Farh, Kyle Kai-How and Fan, Peng-Fei and Yu, Li and Li, Ming and Liu, Zhi-Jin and Tiley, George P. and Yoder, Anne D. and Roos, Christian and Hayakawa, Takashi and Marques-Bonet, Tomas and Rogers, Jeffrey and Stenson, Peter D. and Cooper, David N. and Schierup, Mikkel Heide and Yao, Yong-Gang and Zhang, Ya-Ping and Wang, Wen and Qi, Xiao-Guang and Zhang, Guojie and Wu, Dong-Dong}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Phylogenomic analyses provide insights into primate evolution}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{380}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6648}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{913-924}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1126/science.abn6919}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#C72326"><a href="https://www.science.org/journal/science" rel="external nofollow noopener" target="_blank">Science</a></abbr></div> <div id="rivas-gonzalez2023pervasive" class="col-sm-8"> <div class="title">Pervasive incomplete lineage sorting illuminates speciation and selection in primates</div> <div class="author"> <em>Iker Rivas-González</em>, Marjolaine Rousselle, Fang Li, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Long Zhou, Julien Y. Dutheil, Kasper Munch, Yong Shao, Dongdong Wu, Mikkel H. Schierup, Guojie Zhang' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '0'); ">7 more authors</span> </div> <div class="periodical"> <em>Science</em> 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.science.org/doi/abs/10.1126/science.abn4409" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p> Incomplete lineage sorting (ILS) causes the phylogeny of some parts of the genome to differ from the species tree. In this work, we investigate the frequencies and determinants of ILS in 29 major ancestral nodes across the entire primate phylogeny. We find up to 64% of the genome affected by ILS at individual nodes. We exploit ILS to reconstruct speciation times and ancestral population sizes. Estimated speciation times are much more recent than genomic divergence times and are in good agreement with the fossil record. We show extensive variation of ILS along the genome, mainly driven by recombination but also by the distance to genes, highlighting a major impact of selection on variation along the genome. In many nodes, ILS is reduced more on the X chromosome compared with autosomes than expected under neutrality, which suggests higher impacts of natural selection on the X chromosome. Finally, we show an excess of ILS in genes with immune functions and a deficit of ILS in housekeeping genes. The extensive ILS in primates discovered in this study provides insights into the speciation times, ancestral population sizes, and patterns of natural selection that shape primate evolution. </p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">rivas-gonzalez2023pervasive</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Rivas-González, Iker and Rousselle, Marjolaine and Li, Fang and Zhou, Long and Dutheil, Julien Y. and Munch, Kasper and Shao, Yong and Wu, Dongdong and Schierup, Mikkel H. and Zhang, Guojie}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Pervasive incomplete lineage sorting illuminates speciation and selection in primates}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{380}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6648}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{eabn4409}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1126/science.abn4409}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#2f73e0"><a href="https://joss.theoj.org/" rel="external nofollow noopener" target="_blank">JOSS</a></abbr></div> <div id="rivas-gonzalez2023phasetyper" class="col-sm-8"> <div class="title">PhaseTypeR: an R package for phase-type distributions in population genetics</div> <div class="author"> <em>Iker Rivas-González</em>, Lars Nørvang Andersen, and Asger Hobolth</div> <div class="periodical"> <em>Journal of Open Source Software</em> 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.21105/joss.05054" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p> Phase-type distributions describe the time until absorption of a continuous or discrete-time Markov chain. The probabilistic properties of phase-type distributions (i.e., the probability density function, cumulative distribution function, quantile function, moments and generating functions) are well-described and analytically tractable using matrix manipulations. Phase-type distributions have traditionally been used in actuarial sciences and queuing theory, and, more recently, in population genetics. In order to facilitate the use of phase-type theory in population genetics, we present PhaseTypeR, a general-purpose and user-friendly R package which contains all key functions —mean, (co)variance, probability density function, cumulative distribution function, quantile function and random sampling— for both continuous and discrete phase-type distributions. Importantly, univariate and multivariate reward transformations are implemented for continuous and discrete phase-type distributions. Multivariate reward transformations have great potential for applications in population genetics, and we have included two examples. The first is concerned with the easy calculation of the variance-covariance matrix for the site frequency spectrum (SFS) of the n-coalescent, and the second is concerned with the correlation between tree heights in the two-locus ancestral recombination graph. </p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">rivas-gonzalez2023phasetyper</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Rivas-González, Iker and Andersen, Lars Nørvang and Hobolth, Asger}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{PhaseTypeR}: an {R} package for phase-type distributions in population genetics}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Open Source Software}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{82}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{5054}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{The Open Journal}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.21105/joss.05054}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="year">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#00369f"><a href="https://www.cell.com/" rel="external nofollow noopener" target="_blank">Cell</a></abbr></div> <div id="feng2022incomplete" class="col-sm-8"> <div class="title">Incomplete lineage sorting and phenotypic evolution in marsupials</div> <div class="author"> Shaohong Feng, Ming Bai, <em>Iker Rivas-González</em>, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Cai Li, Shiping Liu, Yijie Tong, Haidong Yang, Guangji Chen, Duo Xie, Karen E Sears, others' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '0'); ">8 more authors</span> </div> <div class="periodical"> <em>Cell</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1016/j.cell.2022.03.034" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p> Incomplete lineage sorting (ILS) makes ancestral genetic polymorphisms persist during rapid speciation events, inducing incongruences between gene trees and species trees. ILS has complicated phylogenetic inference in many lineages, including hominids. However, we lack empirical evidence that ILS leads to incongruent phenotypic variation. Here, we performed phylogenomic analyses to show that the South American monito del monte is the sister lineage of all Australian marsupials, although over 31% of its genome is closer to the Diprotodontia than to other Australian groups due to ILS during ancient radiation. Pervasive conflicting phylogenetic signals across the whole genome are consistent with some of the morphological variation among extant marsupials. We detected hundreds of genes that experienced stochastic fixation during ILS, encoding the same amino acids in non-sister species. Using functional experiments, we confirm how ILS may have directly contributed to hemiplasy in morphological traits that were established during rapid marsupial speciation ca. 60 mya. </p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">feng2022incomplete</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Incomplete lineage sorting and phenotypic evolution in marsupials}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Feng, Shaohong and Bai, Ming and Rivas-Gonz{\'a}lez, Iker and Li, Cai and Liu, Shiping and Tong, Yijie and Yang, Haidong and Chen, Guangji and Xie, Duo and Sears, Karen E and others}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{185}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1646--1660}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.cell.2022.03.034}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2024 Iker Rivas-González. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>